Makefile

```diff
@@ -3,16 +3,25 @@ export GOPATH=/tmp/go
 endif
 
 INSTALL := /usr/bin/install
+DBDIR := /var/run/redis/sonic-db/
 
 all: sonic-telemetry
 
 sonic-telemetry:
```

Comment:
>sonic-telemetry [](start = 0, length = 15)

I am not familiar with golang.
Seems this is a build target, why not use `go build` directly, instead of `go get`. Then we don't need `mkdir` and `cp`

Reply:
go build still require ${GOPATH}/src as source codes workspace, we still need mkdir and cp

Reply:
My understanding is that you can run `go build` without a target, it just compile *.go in current directory.

---
In reply to: [344968232](https://github.com/Azure/sonic-telemetry/pull/24#discussion_r344968232) [](ancestors = 344968232)

Reply:
I did some search online, looks after go version 1.11, go modules are introduced to solve the $GOPATH problem. We can use go mod init and go install to remove the dependency on GOPATH. Also looks our go version in docker is 1.11.5 which is updated this July. Then we build using local files without cp.